src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -122,7 +122,7 @@ pub trait Linker {
     fn optimize(&mut self);
     fn pgo_gen(&mut self);
     fn control_flow_guard(&mut self);
-    fn debuginfo(&mut self);
+    fn strip(&mut self, strip: Strip);
```

Comment:
```suggestion
    fn debuginfo(&mut self, strip: Strip);
```
https://github.com/rust-lang/rust/pull/71825#discussion_r422485981

Reply:
I forgot that, sorry.